decidim-core/lib/decidim/resource_manifest.rb

```diff
@@ -36,6 +36,9 @@ class ResourceManifest
     # The main card to render an instance of the resource.
     attribute :card, String
 
+    # A Hash of the names to use to find linked resources.
+    attribute :link_resource_name, Hash, default: {}
```

Comment:
@oriolgual The intention was to declare it in one place, in order to unify what can be linked to the resource. 
It is done as a `Hash` so that it can be linked to more than one resource, in this PR it introduces `Decidim::Proposals::CollaborativeDraft.resource_manifest.link_resource_name[:proposals]` but in the future it could be added another resource, for example: `Decidim::Proposals::CollaborativeDraft.resource_manifest.link_resource_name[:meetings]`

its used in:
+ [`decidim-proposals/app/cells/decidim/proposals/collaborative_draft_link_to_proposal_cell.rb`](https://github.com/decidim/decidim/pull/3109/files#diff-6be82d86352b89c5558b02c40426d455R16)
+ [`decidim-proposals/app/cells/decidim/proposals/proposal_link_to_collaborative_draft_cell.rb`](https://github.com/decidim/decidim/pull/3109/files#diff-74a98c9806cefe2114bbddfa3a12ac47R16)
+ [`decidim-proposals/app/commands/decidim/proposals/publish_collaborative_draft.rb`](https://github.com/decidim/decidim/pull/3109/files#diff-7e370856447d0eb957929882a9cd12d2R87)

I don't see why you don't see a benefit to it, but If you think it is unuseful  this `attribute :link_resource_name` I'll remove it. :smile:

Reply:
@agustibr but what happens if collaborative drafts (or other resources) are linked in two different ways to proposals? Then you would have:

`Decidim::Proposals::CollaborativeDraft.resource_manifest.link_resource_name[:proposals]`
`Decidim::Proposals::CollaborativeDraft.resource_manifest.link_resource_name[:proposals_with _another_name]`

This means that you'd have to repeat the different keys where you want to use them, and I see no benefit to just use the actual string with the name of the link.

Reply:
Ok, got it. I'll use a string as you suggest with no use of the `resource_manifest` :ok_hand: